include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3590,6 +3593,138 @@ struct OrtApi {
   */
   void(ORT_API_CALL* MemoryInfoGetDeviceType)(_In_ const OrtMemoryInfo* ptr, _Out_ OrtMemoryInfoDeviceType* out);
 
+  /// @}
+  /// \name OrtKernelInfo
+  /// @{
+
+  /** \brief Get the number of inputs from ::OrtKernelInfo.
```

Comment:
It'll be good to mention the scenario(s) where these new APIs will be needed?

Reply:
I added a sentence below this line that mentions potential use in a custom op. Should we include more detail?